src/components/App/index.js

```diff
@@ -56,12 +50,13 @@ function App() {
         requestString={requestString}
         type={type}
       />
-      <AlertWrapper
-        link={responseLink}
-        linkText="Email us"
-        textAfterLink="to respond to a listing"
-        variant="info"
-      />
+      <Button
+        className="mb-3"
+        href={`mailto:${responseEmail}`}
```

Comment:
Does the button component accept an `href`?

Reply:
adding a href prop will automatically render an <a /> element, pretty convenient